src/components/Modal/ModalBasic/ModalMain.tsx

```diff
@@ -0,0 +1,9 @@
+import ModalShareBtns from "./ModalShareBtns";
+
+interface ModalMain {
+  shareButtons?: boolean;
```

Comment:
boolean 데이터는 이름으로 의도를 표현해주면 좋습니다~
```suggestion
  hasShareButtons?: boolean;
```